src/Shell/Task/UnloadTask.php

```diff
@@ -0,0 +1,102 @@
+<?php
+/**
+ * CakePHP(tm) : Rapid Development Framework (http://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * For full copyright and license information, please see the LICENSE.txt
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ * @link          http://cakephp.org CakePHP(tm) Project
+ * @since         3.0.0
+ * @license       http://www.opensource.org/licenses/mit-license.php MIT License
+ */
+namespace Cake\Shell\Task;
+
+use Cake\Console\Shell;
+use Cake\Filesystem\File;
+
+/**
+ * Task for unloading plugins.
+ *
+ */
+class UnloadTask extends Shell
+{
+    /**
+     * Path to the bootstrap file.
+     *
+     * @var string
+     */
+    public $bootstrap = null;
+
+    /**
+     * Execution method always used for tasks.
+     *
+     * @param string $plugin The plugin name.
+     * @return boolean if action passed.
+     *
+     */
+    public function main($plugin = null)
+    {
+        $this->bootstrap = ROOT . DS . 'config' . DS . 'bootstrap.php';
+
+        if (empty($plugin)) {
+            $this->err('<error>You must provide a plugin name in CamelCase format.</error>');
+            $this->err('To unload an "Example" plugin, run <info>`cake plugin unload Example`</info>.');
+            return false;
+        }
+
+
+        $write = $this->_modifyBootstrap($plugin);
+
+        if ($write) {
+            return true;
+        }
+
+        return false;
+    }
+
+    /**
+     * Update the applications bootstrap.php file.
+     *
+     * @param string $plugin Name of plugin.
+     * @return bool If modify passed.
+     */
+    protected function _modifyBootstrap($plugin)
+    {
+        $bool = "(false|true)";
+        $finder = "/Plugin::load\('$plugin', \['autoload' => $bool, 'bootstrap' => $bool, 'routes' => $bool]\);\n/";
```

Comment:
If I add the cleaner code `Plugin::load\('Test'(.)*\);` we are able to support one-line plugin unloading. I think that's better, because when we load a plugin via the shell it will be one line. In a later PR I will add the multiline feature... Agree?